conftest.py

```diff
@@ -45,8 +46,21 @@ def admin():
 
 @pytest.fixture
 def article():
+    en = myapp.config['LANGUAGES'][0][0]
```

Comment:
same question as with `french`

Reply:
and I guess if you're using the language just once you could set it direction at line 53. Unless you might have a good reason.

Reply:
Updated since then.